src/feature/dircache/dirserv.c

```diff
@@ -2679,11 +2676,17 @@ dirserv_read_measured_bandwidths(const char *from_file,
   /* Now would be a nice time to clean the cache, too */
   dirserv_expire_measured_bw_cache(now);
 
-  fclose(fp);
   log_info(LD_DIRSERV,
            "Bandwidth measurement file successfully read. "
            "Applied %d measurements.", applied_lines);
-  return 0;
+  rv = 0;
+
+ err:
+  if (line)
+    raw_free(line);
```

Comment:
Let's add a comment saying that we use raw_free() because getdelim() uses raw_malloc().

Reply:
ok